### ISSUE TITLE
Migrate chat from bare machine to ECS

### ISSUE THREAD

**Issue by rkorytkowski** [OpenConceptLab/ocl_issues#1569]
There is an issue with discourse not starting up properly on ECS due to some write permissions issues to files. It's most likely related to running docker as a different user on ECS and bare machine.

Maybe some adjustments need to be made to docker image created by bootstrap tool from discrouse.

There's the bitnami discourse image, but I'd rather we use the official distribution in case bitnami abandons support.

It's needed for healthchecks and automatic recovery of chat as well as seamless upgrades of host. Right now it always requires manual interventions.

**Comment by paynejd** [comment]
@rkorytkowski Just checking, is this one already completed?

**Comment by rkorytkowski** [comment]
No. It's not high priority at this point. It's just nice to have in terms of future maintenance.
